<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableauDeBord</class>
 <widget class="QWidget" name="TableauDeBord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LaBanque - Tableau De Bord</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="laBanque.qrc">:/ressources/img/LogoLaBanque.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Connecté en tant que</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Marko M.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>-2</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Déconnexion</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dernière connexion le </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>695</x>
      <y>50</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>20/03/2021 à 13h37</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>24</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>44</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Heure</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Début période de réf :</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>48</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>436</x>
      <y>49</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>3 mois</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 mois</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12 mois</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>121</width>
     <height>541</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TBD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>STAT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>AIDE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>REGLAGES</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tableau de bord</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>24</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agence(s) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>56</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Début période :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>86</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chart Format :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>3 mois</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 mois</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12 mois</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="dateEdit_3">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>49</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>86</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Volume</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Euro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pourcentage</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>116</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Appliquer</string>
    </property>
   </widget>
   <widget class="QCheckList" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>13</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>329</y>
     <width>741</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Canaux Détaillés</string>
   </property>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Banque</string>
    </property>
    <widget class="QCustomPlot" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>190</y>
       <width>73</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>190</y>
       <width>73</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>€</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>190</y>
       <width>73</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Assurance</string>
    </property>
    <widget class="QCustomPlot" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>190</y>
       <width>73</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>€</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>190</y>
       <width>73</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>190</y>
       <width>73</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Bourse</string>
    </property>
    <widget class="QCustomPlot" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>190</y>
       <width>73</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>190</y>
       <width>73</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>190</y>
       <width>73</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>€</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>133</y>
     <width>141</width>
     <height>188</height>
    </rect>
   </property>
   <property name="title">
    <string>Chiffre d'Affaires</string>
   </property>
   <widget class="QCustomPlot" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>163</y>
      <width>47</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Vol</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>163</y>
      <width>47</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>€</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_20">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>163</y>
      <width>47</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>132</y>
     <width>221</width>
     <height>188</height>
    </rect>
   </property>
   <property name="title">
    <string>Revenu par Canal</string>
   </property>
   <widget class="QCustomPlot" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_23">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>163</y>
      <width>73</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_22">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>163</y>
      <width>73</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>€</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>163</y>
      <width>73</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Vol</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCheckList</class>
   <extends>QComboBox</extends>
   <header>qchecklist.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="laBanque.qrc"/>
 </resources>
 <connections/>
</ui>
